### PR TITLE
Added ability do unmap volume from host

### PR DESCRIPTION
**Pull request by IlyaKessel**
Added ability to delete volume mapping
image

Dependencies:
https://github.com/ManageIQ/manageiq-api/pull/1075
https://github.com/ManageIQ/manageiq-ui-classic/pull/7862
https://github.com/ManageIQ/manageiq/pull/21432

### CODE REVIEW

**Comment by agrare** [comment]
@IlyaKessel can you please add the list of dependencies / relationships between this PR and any others?  I see a number of other PRs with the similar name but need to know what the merge order needs to be.

**Comment by ereztheodorou** [comment]
@miq-bot add_label morphy/yes?

**Comment by IlyaKessel** [comment]
@agrare I added the list of related PRs to the description

File: app/models/manageiq/providers/autosde/storage_manager/volume_mapping.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
 class ManageIQ::Providers::Autosde::StorageManager::VolumeMapping < ::VolumeMapping
   supports :create
 
+  supports :delete do
+    unsupported_reason_add(:delete, _("The Volume mapping is not connected to an active Manager")) if ext_management_system.nil?
+  end
+
+  def raw_delete_volume_mapping
+    ems = ext_management_system
+    ems.autosde_client.StorageHostVolumeMappingApi.storage_hosts_mapping_pk_delete(ems_ref)
```
Comment (agrare) [inline app/models/manageiq/providers/autosde/storage_manager/volume_mapping.rb:10]: Really minor but in the `raw_create_*` method below you do `ext_management_system.autosde_client...`, best we should stay consistent

Comment (IlyaKessel) [reply app/models/manageiq/providers/autosde/storage_manager/volume_mapping.rb]: @agrare done

**Review by agrare — APPROVED** [review]

**Comment by Fryguy** [comment]
Backported to `morphy` in commit 9c07097cf95bdc15bbedd5cf0011a5c11fdd6eba.

```text
commit 9c07097cf95bdc15bbedd5cf0011a5c11fdd6eba
Author: Adam Grare <adam@grare.com>
Date:   Wed Sep 22 15:21:49 2021 -0400

    Merge pull request #85 from Autosde/volume_mapping_delete
    
    Added ability do unmap volume from host
    
    (cherry picked from commit 3e3d0f7d06b0ccbe095fc719281e2df5a68d3811)
```

**Pull request merged**